custom/opm/beneficiary.py

```diff
@@ -238,7 +238,12 @@ def preg_first_eligible_datetime(self):
             return datetime.datetime.combine(date, datetime.time())
 
     def set_case_properties(self):
-        if self.child_age is None and self.preg_month is None:
+
+        def _is_empty(val):
+            # handle values from FakeConditionsMet as well
```

Comment:
`FakeConditionsMet` is being scrapped off (see [PR](https://github.com/dimagi/commcare-hq/pull/7329/files)) in favour of a `CondtionMet.case_out_of_range` flag. Does this still apply? If not, I think we should revert. (I should have asked this before merging in)